src/Packagist/WebBundle/Resources/views/user/enable_two_factor_auth.html.twig

```diff
@@ -0,0 +1,66 @@
+{% extends "PackagistWebBundle:user:packages.html.twig" %}
+
+{% import "PackagistWebBundle::macros.html.twig" as macros %}
+
+{% set isActualUser = app.user and app.user.username is same as(user.username) %}
+
+{% block content %}
+    <h2 class="title">
+        {{ user.username }}
+        <small>
+            {%- if not isActualUser %}
+                member since: {{ user.createdAt|date('M d, Y') }}
+                {%- if is_granted('ROLE_ADMIN') %}
+                    <a href="mailto:{{ user.email }}">{{ user.email }}</a>
+                {%- endif %}
+            {%- endif %}
+        </small>
+    </h2>
+
+    <section class="row">
+        {% if isActualUser %}
+            <section class="col-md-3">
+                {{ knp_menu_render('profile_menu', {currentClass: 'active', allow_safe_labels: true}) }}
+            </section>
+        {% endif %}
+
+        <section class="{{ isActualUser ? 'col-md-9' : 'col-md-12' }}">
+            <p>
+                To enable two-factor authentication, you'll need a mobile app that supports TOTP such as
+                <a href="https://authy.com/download/">Authy</a>
+                or
+                <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2">Google Authenticator</a>.
+            </p>
+
+            <p><small>(This is not an exhaustive list of compatible apps. Packagist does not endorse or recommend one application over another.)</small></p>
+
+            <img src="{{ qr_code_data_uri(provisioningUri|raw) }}" height="200" class="pull-right" />
```

Comment:
the `raw` filter seems to be in the wrong place. It won't have any effect on the function argument as that's not a place applying auto-escaping. I suggest removing it.

Reply:
Whoops, good catch!  This stray filter has been removed.